src/library/scala/sys/process/package.scala

```diff
@@ -201,12 +201,6 @@ package scala.sys {
     *   associated with it.
     */
   package object process extends ProcessImplicits {
-    /** The arguments passed to `java` when creating this process */
-    def javaVmArguments: List[String] = {
-      import scala.collection.JavaConverters._
-
-      java.lang.management.ManagementFactory.getRuntimeMXBean.getInputArguments.asScala.toList
```

Comment:
Since this method is not deprecated in 2.12, we could do a reflective implementation
```
scala> import scala.collection.JavaConverters._
import scala.collection.JavaConverters._

scala> val mxb = Class.forName("java.lang.management.ManagementFactory").getMethod("getRuntimeMXBean").invoke(null)
mxb: Object = sun.management.RuntimeImpl@4447c594

scala> Class.forName("java.lang.management.RuntimeMXBean").getMethod("getInputArguments").invoke(mxb).asInstanceOf[java.util.List[String]].asScala.toList
res19: List[String] = List(-Xmx1536m, -Xms1536m, -Xss5m, -XX:+UseCompressedOops, -XX:+UseParallelGC, -Xbootclasspath/a:/Users/luc/scala/scala-2.12.3/lib/jline-2.14.4.jar:/Users/luc/scala/scala-2.12.3/lib/scala-compiler.jar:/Users/luc/scala/scala-2.12.3/lib/scala-library.jar:/Users/luc/scala/scala-2.12.3/lib/scala-parser-combinators_2.12-1.0.6.jar:/Users/luc/scala/scala-2.12.3/lib/scala-reflect.jar:/Users/luc/scala/scala-2.12.3/lib/scala-swing_2.12-2.0.0.jar:/Users/luc/scala/scala-2.12.3/lib/scala-xml_2.12-1.0.6.jar:/Users/luc/scala/scala-2.12.3/lib/scalap-2.12.3.jar, -Dscala.boot.class.path=/Users/luc/scala/scala-2.12.3/lib/jline-2.14.4.jar:/Users/luc/scala/scala-2.12.3/lib/scala-compiler.jar:/Users/luc/scala/scala-2.12.3/lib/scala-library.jar:/Users/luc/scala/...
```

This would fail with `java.lang.ClassNotFoundException` if the classes are not available.

We can deprecate the method at the same time.

Reply:
I see you're deprecating it for 2.12.5; I guess either way is fine.